File: Digital Business Enablement/bluegather/src/main/java/br/com/fiap/bluegather/dto/ImagemDTO.java
package br.com.fiap.bluegather.dto;

import jakarta.validation.constraints.*;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ImagemDTO {
    private Long id;
    
    @NotNull(message = "O campo idEvento não pode estar vazio.")
    private Long idEvento;

    @NotNull(message = "O campo idMomento não pode estar vazio.")
    private Long idMomento;

    @NotNull(message = "O campo urlImagem não pode estar vazio.")
    private String urlImagem;
}

File: Enterprise Application Development/bluegather/Dtos/ImagemDto.cs
using System.ComponentModel.DataAnnotations;

namespace BlueGather.Dto
{
    public class ImagemDto
    {
        public long? Id { get; set; }

        [Required(ErrorMessage = "O campo idEvento não pode estar vazio.")]
        public long IdEvento { get; set; }

        [Required(ErrorMessage = "O campo idMomento não pode estar vazio.")]
        public long IdMomento { get; set; }

        [Required(ErrorMessage = "O campo urlImagem não pode estar vazio.")]
        public string UrlImagem { get; set; }
    }
}